src/test/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCountsIntegrationTest.java

```diff
@@ -98,7 +96,7 @@ public void testBadTargetFile() throws IOException {
         runCommandLine(arguments);
     }
 
-    @Test(expectedExceptions = UserException.BadInput.class)
+    @Test
```

Comment:
This one took me a while to figure out: the name of the test is misleading.  It wasn't supposed to fail due to the extra target (targets not in the PoN are ignored.  Note that the next test, `testFullInputWithExtraTargetOneSample` does not have an expected exception.  It failed because it had more than one sample, because converting to `TargetCoverage` requires a single sample.  But now that output remains a `ReadCountCollection` there is no problem with multiple samples.